src/business_logic/state/mod.rs

```diff
@@ -3,6 +3,9 @@ pub(crate) mod contract_storage_state;
 pub mod state_api;
 pub mod state_cache;
 
+pub mod in_memory_state_reader;
+pub mod structs;
```

Comment:
can you move the StateDiff and the ExecutionResourcesManager to different files to avoid having a `structs::` prefix? Or move them to state/mod.rs so we can import them directly from `state`